src/Controls/src/Core/HandlerImpl/ControlsWindow.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.Maui;
+
+namespace Microsoft.Maui.Controls
+{
+	public class Window : VisualElement, IWindow
```

Comment:
Is not important, but, could we use the same naming used with the rest: `Window.Impl.cs`

Reply:
done